pkg/domain/model/message_model.go

```diff
@@ -0,0 +1,22 @@
+package model
+
+import "time"
+
+type MType int
+
+const (
+	Txt MType = iota + 1
+	Remind
+	ImageType
```

Comment:
```suggestion
	Text MType = iota + 1
	Remind
	Image
```
ここImageだけTypeあるのおかしかったわごめんレビュウミス